packages/plugin-kafkajs/src/kafkajs.ts

```diff
@@ -22,12 +14,17 @@ import {
     EachMessagePayload,
     KafkaMessage,
     EachBatchPayload,
-    IHeaders,
     Consumer,
 } from 'kafkajs';
 import { KafkaJsPluginConfig } from './types';
-import { AttributeNames } from './enums';
 import { VERSION } from './version';
+import { bufferTextMapGetter } from './propagtor';
+
+enum MessagingOperationType {
```

Comment:
https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-semantic-conventions/src/trace/messaging.ts#L136 
it's already exported from semantic-conventions